dissect/target/target.py

```diff
@@ -482,6 +483,20 @@ def _init_os(self) -> None:
         self._os_plugin = os_plugin
         self._os = self.add_plugin(os_plugin.create(self, fs))
 
+    def _mount_others(self) -> None:
+        root_fs = self.fs
+        counter = 0
+        path = "/$fs$/fs0"
+
+        for fs in self.filesystems:
+            # determine mount point
+            while root_fs.path(path).exists():
+                counter += 1
+                path = f"/$fs$/fs{counter}"
```

Comment:
Wouldn't it be more logical/efficient to have this while loop inside the `if` below?